Dịch "aws_vn" - Phần 3 (#3005)
aivivn/d2l-vn: chapter_appendix-tools-for-deep-learning/aws_vn.md

### Connecting to the Instance
-->

### Kết nối tới Máy ảo

<!--
As shown in :numref:`fig_connect`, after the instance state turns green, 
right-click the instance and select `Connect` to view the instance access method.
-->

Như đã chỉ ra trong :numref:`fig_connect`, sau khi trạng thái máy ảo chuyển màu xanh,

> nhấp chuột phải vào máy ảo và chọn `Kết nối - Connect` để quan sát phương thức truy cập máy ảo.

Reviewer: ```suggestion
nhấp chuột phải vào máy ảo và chọn `Connect` để quan sát phương thức truy cập máy ảo.
```

Now, copy the ssh command in the lower red box of :numref:`fig_chmod` and paste onto the command line:
-->

> Giờ hãy sao chép lệnh ssh trong khung đỏ phía dưới trong :numref:`fig_chmod` và dán vào command line:

Reviewer: ```suggestion
Giờ hãy sao chép lệnh ssh trong khung đỏ phía dưới trong :numref:`fig_chmod` và dán vào cửa sổ dòng lệnh:
```
từ này có trong glossary rồi.

When the command line prompts "Are you sure you want to continue connecting (yes/no)", enter "yes" and press Enter to log into the instance.
-->

> Khi command line thông báo "Are you sure you want to continue connecting (yes/no)", nhập "yes" và nhấn Enter để đăng nhập vào máy ảo.

Reviewer: ```suggestion
Khi cửa sổ dòng lệnh thông báo "Are you sure you want to continue connecting (yes/no)", nhập "yes" và nhấn Enter để đăng nhập vào máy ảo.
```

Copy the instructions and paste them into the terminal to install CUDA 10.1.
-->

> Sao chép các lệnh và dán vào terminal để cài đặt CUDA 10.1.

Reviewer: ```suggestion
Sao chép các lệnh và dán vào cửa sổ dòng lệnh để cài đặt CUDA 10.1.
```

## Installing MXNet and Downloading the D2L Notebooks
-->

## Cài đặt MXNet và Tải Notebook của D2L

<!--
First, to simplify the installation, you need to install [Miniconda](https://conda.io/en/latest/miniconda.html) for Linux.
The download link and file name are subject to changes, so please go the Miniconda website and click "Copy Link Address" as shown in :numref:`fig_miniconda`.
-->

Đầu tiên, để đơn giản hoá quá trình cài đặt, bạn cần cài đặt [Miniconda](https://conda.io/en/latest/miniconda.html) cho Linux.

> Đường dẫn tải về và tên tệp có thể thay đổi, vậy nên vui lòng truy cập trang web Miniconda và chọn "Sao chép đường dẫn tải về - Copy Link Address" như chỉ ra trong :numref:`fig_miniconda`.

Reviewer: ```suggestion
Đường dẫn tải về và tên tệp có thể thay đổi, vậy nên vui lòng truy cập trang web Miniconda và chọn "Copy Link Address" như chỉ ra trong :numref:`fig_miniconda`.
```
